GameData/RealismOverhaul/RO_SuggestedMods/Bluedog_DB/RO_BDB_Pioneer_NEW.cfg

```diff
@@ -0,0 +1,453 @@
+	@manufacturer = General Electric
+	%skinMaxTemp = 873.15
+	%crashTolerance = 12
+	%breakingForce = 250
+	%breakingTorque = 250
+}
+
+@PART[bluedog_Pioneer_Radiator]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	%rescaleFactor = 1.6
+	@mass = 0.0015
+	@maxTemp = 773.15
+	@title = Pioneer 10/11 Shunt Radiator
+	@manufacturer = TRW
+	@description = A small shunt radiator designed for the Pioneer 10/11 spacecraft to aid in maintaining thermal control.
+	%skinMaxTemp = 873.15
+	%crashTolerance = 12
+	%breakingForce = 250
+	%breakingTorque = 250
+}
+
+@PART[bluedog_Pioneer_StellarReferenceAssembly]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	%rescaleFactor = 1.6
+	@mass = 0.0015
+	@maxTemp = 773.15
+	@title = Pioneer 10/11 Stellar Reference Assembly
+	@manufacturer = TRW
+	@description = A lightweight star tracker designed to use the star Canopus as a reference point. Even though designed to help track the position, attitude, and spin of the Pioneer 10/11 spacecraft, this star tracker can be used to provide additional SAS control modes to other probe cores as well.
+	%skinMaxTemp = 873.15
+	%crashTolerance = 12
+	%breakingForce = 250
+	%breakingTorque = 250
+}
+
+@PART[bluedog_Pioneer_IPP]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	%rescaleFactor = 1.6
+	@mass = 0.001
+	@maxTemp = 773.15
+	@title = Pioneer 10/11 Imaging Photopolarimeter
+	@manufacturer = University of Arizona
+	@description = A small photopolarimeter imaging system intended for use with spin-stabilized spacecraft and designed for the Pioneer 10/11 spacecraft. Unlike traditional TV camera imaging, this instrument does not have a vidicon tube, and instead uses spin-scan imaging techniques. It can also preform photopolarimetric observations in addition to imaging. Despite its crude and primitive spin-scan imagery it is highly robust and works in the harshest environments. Includes both a basic imaging experiment and a photopolarimeter experiment.
+	%skinMaxTemp = 873.15
+	%crashTolerance = 12
+	%breakingForce = 250
+	%breakingTorque = 250
+}
+
+@PART[bluedog_Pioneer_LGA]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	%rescaleFactor = 1.6
+	@mass = 0.001
+	@maxTemp = 773.15
+	@title = Pioneer 10/11 Low Gain Omni Antenna
+	@manufacturer = TWR
+	@description = A small, fixed omnidirectional s-band low gain antenna. Designed for the Pioneer 10/11 spacecraft, this antenna is designed to communicate with ground stations during launch and the beginning of the its cruise period. While designed as a secondary antenna, this antenna could be used for any number of applications, including as a primary antenna.
+	%skinMaxTemp = 873.15
+	%crashTolerance = 12
+	%breakingForce = 250
+	%breakingTorque = 250
+}
+
+@PART[bluedog_Pioneer_lGA]:BEFORE[RealAntennas]:NEEDS[RealAntennas]
+{
+	!MODULE[ModuleDataTransmitter],* {}
+	MODULE
+	{
+		name = ModuleRealAntenna
+		referenceGain = 4.0		//Based on size and construction
+		RFBand = S		//Default to S-band
+	}
+}

```

Comment:
I would suggest no higher than 3dBi of gain unless you can cite a data sheet.
Also fix typo in the part name [capitalization]
BEFORE implies NEEDS, so you can delete the NEEDS portion.